Assets/MRTK/Providers/LeapMotion/Editor/ConfigurationChecker/LeapMotionConfigurationChecker.cs

```diff
@@ -0,0 +1,263 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.LeapMotion
+{
+    /// <summary>
+    /// Class that checks if the Leap Motion Core assets are present and configures the project if they are.
+    /// </summary>
+    [InitializeOnLoad]
+    static class LeapMotionConfigurationChecker
+    {
+        private const string FileName = "README_BEFORE_UPDATING.txt";
+        private static readonly string[] definitions = { "LEAPMOTIONCORE_PRESENT" };
```

Comment:
@keveleigh is it "Definitions" or "defintions"? since it's effectively a const.

Reply:
Yeah, I think we should use PascalCase for `static readonly`

[StyleCop agrees](https://github.com/StyleCop/StyleCop/blob/c2b30c50c4cccd963fc74525b0bdf01c74f6d831/Project/Docs/Rules/SA1311.html#L26-L27) 😜